### ISSUE TITLE
Build failure on Debian armel

### ISSUE THREAD

**Issue by sbrun** [aircrack-ng/aircrack-ng#1872]
Hello,
We packaged latest released version in Debian but it fails to build on armel
we use ```configure --with-gcrypt --with-experimental```

First try without openssl:
```
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I. -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -mfpu=neon -DSIMD_CORE -DHAS_NEON -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__neon-crypto.o `test -f 'crypto.c' || echo './'`crypto.c
In file included from crypto.h:40:0,
                 from crypto.c:40:
crypto.c: In function 'calc_mic':
gcrypt-openssl-wrapper.h:63:65: warning: 'ctx' may be used uninitialized in this function [-Wmaybe-uninitialized]
                                                                 gcry_md_reset(*ctx); \
                                                                 ^~~~~~~~~~~~~
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I. -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -mfpu=neon -DSIMD_CORE -DHAS_NEON -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__neon-wpapsk.o `test -f 'wpapsk.c' || echo './'`wpapsk.c
wpapsk.c: In function 'wpapsk_sse':
wpapsk.c:223:35: error: 'SIMD_COEF_32' undeclared (first use in this function); did you mean 'SIMD_CORE'?
    ((unsigned int *)sse_hash1)[15*SIMD_COEF_32 + (index&(SIMD_COEF_32-1)) + (unsigned int)index/SIMD_COEF_32*SHA_BUF_SIZ*SIMD_COEF_32] = (84<<3); // all encrypts are 64+20 bytes.
                                   ^~~~~~~~~~~~
                                   SIMD_CORE
wpapsk.c:223:35: note: each undeclared identifier is reported only once for each function it appears in
wpapsk.c:223:110: error: 'SHA_BUF_SIZ' undeclared (first use in this function); did you mean '_G_BUFSIZ'?
    ((unsigned int *)sse_hash1)[15*SIMD_COEF_32 + (index&(SIMD_COEF_32-1)) + (unsigned int)index/SIMD_COEF_32*SHA_BUF_SIZ*SIMD_COEF_32] = (84<<3); // all encrypts are 64+20 bytes.
                                                                                                              ^~~~~~~~~~~
                                                                                                              _G_BUFSIZ
wpapsk.c:335:4: warning: implicit declaration of function 'SIMDSHA1body' [-Wimplicit-function-declaration]
    SIMDSHA1body((unsigned int*)t_sse_hash1, (unsigned int*)t_sse_hash1, (unsigned int*)t_sse_crypt1, SSEi_MIXED_IN|SSEi_RELOAD|SSEi_OUTPUT_AS_INP_FMT);
    ^~~~~~~~~~~~
Makefile:2747: recipe for target 'aircrack_ng__neon-wpapsk.o' failed
make[3]: *** [aircrack_ng__neon-wpapsk.o] Error 1
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[3]: *** Waiting for unfinished jobs....
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I. -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -mfpu=neon -DSIMD_CORE -DHAS_NEON -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__neon-uniqueiv.o `test -f 'uniqueiv.c' || echo './'`uniqueiv.c
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -DOLD_SSE_CORE=1 -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__generic-aircrack-ng.o `test -f 'aircrack-ng.c' || echo './'`aircrack-ng.c
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I. -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -mfpu=neon -DSIMD_CORE -DHAS_NEON -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__neon-aircrack-ng.o `test -f 'aircrack-ng.c' || echo './'`aircrack-ng.c
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
Makefile:3616: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
Makefile:557: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
dh_auto_build: make -j4 -Oline returned exit code 2
debian/rules:6: recipe for target 'build-arch' failed
```
Complete log can be found here:
https://buildd.debian.org/status/fetch.php?pkg=aircrack-ng&arch=armel&ver=1%3A1.2.0-1&stamp=1525240160&raw=0

And a second try with openssl in build-depends (same error):
```
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I. -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -mfpu=neon -DSIMD_CORE -DHAS_NEON -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__neon-crypto.o `test -f 'crypto.c' || echo './'`crypto.c
In file included from crypto.h:40:0,
                 from crypto.c:40:
crypto.c: In function 'calc_mic':
gcrypt-openssl-wrapper.h:63:65: warning: 'ctx' may be used uninitialized in this function [-Wmaybe-uninitialized]
                                                                 gcry_md_reset(*ctx); \
                                                                 ^~~~~~~~~~~~~
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I. -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -mfpu=neon -DSIMD_CORE -DHAS_NEON -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__neon-uniqueiv.o `test -f 'uniqueiv.c' || echo './'`uniqueiv.c
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I. -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -mfpu=neon -DSIMD_CORE -DHAS_NEON -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__neon-wpapsk.o `test -f 'wpapsk.c' || echo './'`wpapsk.c
wpapsk.c: In function 'wpapsk_sse':
wpapsk.c:223:35: error: 'SIMD_COEF_32' undeclared (first use in this function); did you mean 'SIMD_CORE'?
    ((unsigned int *)sse_hash1)[15*SIMD_COEF_32 + (index&(SIMD_COEF_32-1)) + (unsigned int)index/SIMD_COEF_32*SHA_BUF_SIZ*SIMD_COEF_32] = (84<<3); // all encrypts are 64+20 bytes.
                                   ^~~~~~~~~~~~
                                   SIMD_CORE
wpapsk.c:223:35: note: each undeclared identifier is reported only once for each function it appears in
wpapsk.c:223:110: error: 'SHA_BUF_SIZ' undeclared (first use in this function); did you mean '_G_BUFSIZ'?
    ((unsigned int *)sse_hash1)[15*SIMD_COEF_32 + (index&(SIMD_COEF_32-1)) + (unsigned int)index/SIMD_COEF_32*SHA_BUF_SIZ*SIMD_COEF_32] = (84<<3); // all encrypts are 64+20 bytes.
                                                                                                              ^~~~~~~~~~~
                                                                                                              _G_BUFSIZ
wpapsk.c:335:4: warning: implicit declaration of function 'SIMDSHA1body' [-Wimplicit-function-declaration]
    SIMDSHA1body((unsigned int*)t_sse_hash1, (unsigned int*)t_sse_hash1, (unsigned int*)t_sse_crypt1, SSEi_MIXED_IN|SSEi_RELOAD|SSEi_OUTPUT_AS_INP_FMT);
    ^~~~~~~~~~~~
Makefile:2747: recipe for target 'aircrack_ng__neon-wpapsk.o' failed
make[3]: *** [aircrack_ng__neon-wpapsk.o] Error 1
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[3]: *** Waiting for unfinished jobs....
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I. -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE  -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__neon-sha1-sse2.o `test -f 'sha1-sse2.S' || echo './'`sha1-sse2.S
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -DOLD_SSE_CORE=1 -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__generic-aircrack-ng.o `test -f 'aircrack-ng.c' || echo './'`aircrack-ng.c
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[3]: Entering directory '/<<PKGBUILDDIR>>/src'
gcc -DPACKAGE_NAME=\"aircrack-ng\" -DPACKAGE_TARNAME=\"aircrack-ng\" -DPACKAGE_VERSION=\"1.2\" -DPACKAGE_STRING=\"aircrack-ng\ 1.2\" -DPACKAGE_BUGREPORT=\"https://forum.aircrack-ng.org\" -DPACKAGE_URL=\"\" -DPACKAGE=\"aircrack-ng\" -DVERSION=\"1.2\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -D_REVISION=\"1.2\" -DHAVE_GCRYPT=1 -DGCRYPT_WITH_AES=1 -DGCRYPT_WITH_ARCFOUR=1 -DGCRYPT_WITH_BLOWFISH=1 -DGCRYPT_WITH_CAST5=1 -DGCRYPT_WITH_DES=1 -DGCRYPT_WITH_IDEA=1 -DGCRYPT_WITH_RFC2268=1 -DGCRYPT_WITH_SERPENT=1 -DGCRYPT_WITH_TWOFISH=1 -DGCRYPT_WITH_CRC=1 -DGCRYPT_WITH_MD4=1 -DGCRYPT_WITH_MD5=1 -DGCRYPT_WITH_RMD160=1 -DGCRYPT_WITH_SHA1=1 -DGCRYPT_WITH_SHA256=1 -DGCRYPT_WITH_SHA512=1 -DGCRYPT_WITH_TIGER=1 -DGCRYPT_WITH_WHIRLPOOL=1 -DGCRYPT_WITH_DSA=1 -DGCRYPT_WITH_ELGAMAL=1 -DGCRYPT_WITH_RSA=1 -DHAVE_PCAP_H=1 -DHAVE_PCAP=1 -DHAVE_PCRE=1 -DHAVE_SQLITE3_H=1 -DHAVE_SQLITE3=1 -DHAVE_SQLITE=1 -DHAVE_ZLIB=1 -D_FILE_OFFSET_BITS=64 -DHAVE_PTHREAD_PRIO_INHERIT=1 -D_REENTRANT=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -DHAVE_SYS_AUXV_H=1 -DHAS_AUXV=1 -I.  -I. -I.. -I../src/include  -Wdate-time -D_FORTIFY_SOURCE=2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -mfpu=neon -DSIMD_CORE -DHAS_NEON -pthread -DUSE_GCRYPT  -I/usr/include -Wall -O3 -std=gnu99 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-array-bounds -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -c -o aircrack_ng__neon-aircrack-ng.o `test -f 'aircrack-ng.c' || echo './'`aircrack-ng.c
make[3]: Leaving directory '/<<PKGBUILDDIR>>/src'
Makefile:3616: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
Makefile:557: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
dh_auto_build: make -j4 -Oline returned exit code 2
debian/rules:6: recipe for target 'build-arch' failed
make: *** [build-arch] Error 25
```
Complete log can be found here:
https://buildd.debian.org/status/fetch.php?pkg=aircrack-ng&arch=armel&ver=1%3A1.2.0-2&stamp=1525286148&raw=0

**Comment by aircrack-ng** [comment]
Could you run `gcc -dM -E - < /dev/null` and paste the result here?

**Comment by LocutusOfBorg** [comment]
```
(sid_armel-dchroot)locutusofborg@abel:~$ gcc -dM -E - < /dev/null
#define __DBL_MIN_EXP__ (-1021)
#define __HQ_FBIT__ 15
#define __FLT32X_MAX_EXP__ 1024
#define __UINT_LEAST16_MAX__ 0xffff
#define __ARM_SIZEOF_WCHAR_T 4
#define __ATOMIC_ACQUIRE 2
#define __SFRACT_IBIT__ 0
#define __FLT_MIN__ 1.1754943508222875e-38F
#define __GCC_IEC_559_COMPLEX 0
#define __UFRACT_MAX__ 0XFFFFP-16UR
#define __UINT_LEAST8_TYPE__ unsigned char
#define __DQ_FBIT__ 63
#define __INTMAX_C(c) c ## LL
#define __ULFRACT_FBIT__ 32
#define __SACCUM_EPSILON__ 0x1P-7HK
#define __CHAR_BIT__ 8
#define __USQ_IBIT__ 0
#define __UINT8_MAX__ 0xff
#define __ACCUM_FBIT__ 15
#define __WINT_MAX__ 0xffffffffU
#define __FLT32_MIN_EXP__ (-125)
#define __USFRACT_FBIT__ 8
#define __ORDER_LITTLE_ENDIAN__ 1234
#define __SIZE_MAX__ 0xffffffffU
#define __ARM_ARCH_ISA_ARM 1
#define __WCHAR_MAX__ 0xffffffffU
#define __LACCUM_IBIT__ 32
#define __DBL_DENORM_MIN__ ((double)4.9406564584124654e-324L)
#define __GCC_ATOMIC_CHAR_LOCK_FREE 1
#define __GCC_IEC_559 0
#define __FLT32X_DECIMAL_DIG__ 17
#define __FLT_EVAL_METHOD__ 0
#define __unix__ 1
#define __LLACCUM_MAX__ 0X7FFFFFFFFFFFFFFFP-31LLK
#define __FLT64_DECIMAL_DIG__ 17
#define __GCC_ATOMIC_CHAR32_T_LOCK_FREE 1
#define __FRACT_FBIT__ 15
#define __UINT_FAST64_MAX__ 0xffffffffffffffffULL
#define __SIG_ATOMIC_TYPE__ int
#define __UACCUM_FBIT__ 16
#define __DBL_MIN_10_EXP__ (-307)
#define __FINITE_MATH_ONLY__ 0
#define __ARMEL__ 1
#define __LFRACT_IBIT__ 0
#define __GNUC_PATCHLEVEL__ 0
#define __FLT32_HAS_DENORM__ 1
#define __LFRACT_MAX__ 0X7FFFFFFFP-31LR
#define __UINT_FAST8_MAX__ 0xff
#define __has_include(STR) __has_include__(STR)
#define __DEC64_MAX_EXP__ 385
#define __INT8_C(c) c
#define __INT_LEAST8_WIDTH__ 8
#define __UINT_LEAST64_MAX__ 0xffffffffffffffffULL
#define __SA_FBIT__ 15
#define __SHRT_MAX__ 0x7fff
#define __LDBL_MAX__ 1.7976931348623157e+308L
#define __FRACT_MAX__ 0X7FFFP-15R
#define __ARM_ARCH_5TE__ 1
#define __UFRACT_FBIT__ 16
#define __UFRACT_MIN__ 0.0UR
#define __UINT_LEAST8_MAX__ 0xff
#define __GCC_ATOMIC_BOOL_LOCK_FREE 1
#define __UINTMAX_TYPE__ long long unsigned int
#define __LLFRACT_EPSILON__ 0x1P-63LLR
#define __linux 1
#define __DEC32_EPSILON__ 1E-6DF
#define __FLT_EVAL_METHOD_TS_18661_3__ 0
#define __CHAR_UNSIGNED__ 1
#define __UINT32_MAX__ 0xffffffffU
#define __ULFRACT_MAX__ 0XFFFFFFFFP-32ULR
#define __TA_IBIT__ 64
#define __LDBL_MAX_EXP__ 1024
#define __WINT_MIN__ 0U
#define __linux__ 1
#define __INT_LEAST16_WIDTH__ 16
#define __ULLFRACT_MIN__ 0.0ULLR
#define __SCHAR_MAX__ 0x7f
#define __WCHAR_MIN__ 0U
#define __INT64_C(c) c ## LL
#define __DBL_DIG__ 15
#define __GCC_ATOMIC_POINTER_LOCK_FREE 1
#define __LLACCUM_MIN__ (-0X1P31LLK-0X1P31LLK)
#define __SIZEOF_INT__ 4
#define __SIZEOF_POINTER__ 4
#define __USACCUM_IBIT__ 8
#define __USER_LABEL_PREFIX__ 
#define __STDC_HOSTED__ 1
#define __LDBL_HAS_INFINITY__ 1
#define __LFRACT_MIN__ (-0.5LR-0.5LR)
#define __HA_IBIT__ 8
#define __FLT32_DIG__ 6
#define __TQ_IBIT__ 0
#define __FLT_EPSILON__ 1.1920928955078125e-7F
#define __APCS_32__ 1
#define __SHRT_WIDTH__ 16
#define __USFRACT_IBIT__ 0
#define __LDBL_MIN__ 2.2250738585072014e-308L
#define __STDC_UTF_16__ 1
#define __FRACT_MIN__ (-0.5R-0.5R)
#define __DEC32_MAX__ 9.999999E96DF
#define __DA_IBIT__ 32
#define __ARM_SIZEOF_MINIMAL_ENUM 4
#define __FLT32X_HAS_INFINITY__ 1
#define __INT32_MAX__ 0x7fffffff
#define __UQQ_FBIT__ 8
#define __INT_WIDTH__ 32
#define __SIZEOF_LONG__ 4
#define __UACCUM_MAX__ 0XFFFFFFFFP-16UK
#define __STDC_ISO_10646__ 201706L
#define __UINT16_C(c) c
#define __PTRDIFF_WIDTH__ 32
#define __DECIMAL_DIG__ 17
#define __LFRACT_EPSILON__ 0x1P-31LR
#define __FLT64_EPSILON__ 2.2204460492503131e-16F64
#define __ULFRACT_MIN__ 0.0ULR
#define __gnu_linux__ 1
#define __INTMAX_WIDTH__ 64
#define __has_include_next(STR) __has_include_next__(STR)
#define __LDBL_HAS_QUIET_NAN__ 1
#define __ULACCUM_IBIT__ 32
#define __FLT64_MANT_DIG__ 53
#define __UACCUM_EPSILON__ 0x1P-16UK
#define __GNUC__ 7
#define __ULLACCUM_MAX__ 0XFFFFFFFFFFFFFFFFP-32ULLK
#define __pie__ 2
#define __HQ_IBIT__ 0
#define __FLT_HAS_DENORM__ 1
#define __SIZEOF_LONG_DOUBLE__ 8
#define __BIGGEST_ALIGNMENT__ 8
#define __FLT64_MAX_10_EXP__ 308
#define __GNUC_STDC_INLINE__ 1
#define __DQ_IBIT__ 0
#define __DBL_MAX__ ((double)1.7976931348623157e+308L)
#define __ULFRACT_IBIT__ 0
#define __INT_FAST32_MAX__ 0x7fffffff
#define __DBL_HAS_INFINITY__ 1
#define __ACCUM_IBIT__ 16
#define __DEC32_MIN_EXP__ (-94)
#define __THUMB_INTERWORK__ 1
#define __INTPTR_WIDTH__ 32
#define __LACCUM_MAX__ 0X7FFFFFFFFFFFFFFFP-31LK
#define __FLT32X_HAS_DENORM__ 1
#define __INT_FAST16_TYPE__ int
#define __LDBL_HAS_DENORM__ 1
#define __DEC128_MAX__ 9.999999999999999999999999999999999E6144DL
#define __INT_LEAST32_MAX__ 0x7fffffff
#define __ARM_PCS 1
#define __DEC32_MIN__ 1E-95DF
#define __ACCUM_MAX__ 0X7FFFFFFFP-15K
#define __DBL_MAX_EXP__ 1024
#define __USACCUM_EPSILON__ 0x1P-8UHK
#define __WCHAR_WIDTH__ 32
#define __FLT32_MAX__ 3.4028234663852886e+38F32
#define __DEC128_EPSILON__ 1E-33DL
#define __SFRACT_MAX__ 0X7FP-7HR
#define __FRACT_IBIT__ 0
#define __PTRDIFF_MAX__ 0x7fffffff
#define __UACCUM_MIN__ 0.0UK
#define __STDC_NO_THREADS__ 1
#define __UACCUM_IBIT__ 16
#define __FLT32_HAS_QUIET_NAN__ 1
#define __LONG_LONG_MAX__ 0x7fffffffffffffffLL
#define __SIZEOF_SIZE_T__ 4
#define __ULACCUM_MAX__ 0XFFFFFFFFFFFFFFFFP-32ULK
#define __SIZEOF_WINT_T__ 4
#define __LONG_LONG_WIDTH__ 64
#define __FLT32_MAX_EXP__ 128
#define __SA_IBIT__ 16
#define __ULLACCUM_MIN__ 0.0ULLK
#define __GXX_ABI_VERSION 1011
#define __UTA_FBIT__ 64
#define __SOFTFP__ 1
#define __FLT_MIN_EXP__ (-125)
#define __USFRACT_MAX__ 0XFFP-8UHR
#define __UFRACT_IBIT__ 0
#define __ARM_FEATURE_QBIT 1
#define __INT_FAST64_TYPE__ long long int
#define __FLT64_DENORM_MIN__ 4.9406564584124654e-324F64
#define __DBL_MIN__ ((double)2.2250738585072014e-308L)
#define __PIE__ 2
#define __FLT32X_EPSILON__ 2.2204460492503131e-16F32x
#define __FLT64_MIN_EXP__ (-1021)
#define __LACCUM_MIN__ (-0X1P31LK-0X1P31LK)
#define __ULLACCUM_FBIT__ 32
#define __GXX_TYPEINFO_EQUALITY_INLINE 0
#define __FLT64_MIN_10_EXP__ (-307)
#define __ULLFRACT_EPSILON__ 0x1P-64ULLR
#define __DEC128_MIN__ 1E-6143DL
#define __REGISTER_PREFIX__ 
#define __UINT16_MAX__ 0xffff
#define __DBL_HAS_DENORM__ 1
#define __ACCUM_MIN__ (-0X1P15K-0X1P15K)
#define __SQ_IBIT__ 0
#define __FLT32_MIN__ 1.1754943508222875e-38F32
#define __UINT8_TYPE__ unsigned char
#define __UHA_FBIT__ 8
#define __NO_INLINE__ 1
#define __SFRACT_MIN__ (-0.5HR-0.5HR)
#define __UTQ_FBIT__ 128
#define __FLT_MANT_DIG__ 24
#define __LDBL_DECIMAL_DIG__ 17
#define __VERSION__ "7.3.0"
#define __UINT64_C(c) c ## ULL
#define __ULLFRACT_FBIT__ 64
#define __FRACT_EPSILON__ 0x1P-15R
#define __ULACCUM_MIN__ 0.0ULK
#define _STDC_PREDEF_H 1
#define __UDA_FBIT__ 32
#define __LLACCUM_EPSILON__ 0x1P-31LLK
#define __GCC_ATOMIC_INT_LOCK_FREE 1
#define __FLT32_MANT_DIG__ 24
#define __FLOAT_WORD_ORDER__ __ORDER_LITTLE_ENDIAN__
#define __USFRACT_MIN__ 0.0UHR
#define __UQQ_IBIT__ 0
#define __SCHAR_WIDTH__ 8
#define __INT32_C(c) c
#define __DEC64_EPSILON__ 1E-15DD
#define __ORDER_PDP_ENDIAN__ 3412
#define __DEC128_MIN_EXP__ (-6142)
#define __UHQ_FBIT__ 16
#define __LLACCUM_FBIT__ 31
#define __FLT32_MAX_10_EXP__ 38
#define __INT_FAST32_TYPE__ int
#define __UINT_LEAST16_TYPE__ short unsigned int
#define unix 1
#define __INT16_MAX__ 0x7fff
#define __SIZE_TYPE__ unsigned int
#define __UINT64_MAX__ 0xffffffffffffffffULL
#define __UDQ_FBIT__ 64
#define __INT8_TYPE__ signed char
#define __ELF__ 1
#define __ULFRACT_EPSILON__ 0x1P-32ULR
#define __LLFRACT_FBIT__ 63
#define __FLT_RADIX__ 2
#define __INT_LEAST16_TYPE__ short int
#define __LDBL_EPSILON__ 2.2204460492503131e-16L
#define __UINTMAX_C(c) c ## ULL
#define __SACCUM_MAX__ 0X7FFFP-7HK
#define __SIG_ATOMIC_MAX__ 0x7fffffff
#define __GCC_ATOMIC_WCHAR_T_LOCK_FREE 1
#define __VFP_FP__ 1
#define __SIZEOF_PTRDIFF_T__ 4
#define __FLT32X_MANT_DIG__ 53
#define __LACCUM_EPSILON__ 0x1P-31LK
#define __FLT32X_MIN_EXP__ (-1021)
#define __DEC32_SUBNORMAL_MIN__ 0.000001E-95DF
#define __INT_FAST16_MAX__ 0x7fffffff
#define __FLT64_DIG__ 15
#define __UINT_FAST32_MAX__ 0xffffffffU
#define __UINT_LEAST64_TYPE__ long long unsigned int
#define __USACCUM_MAX__ 0XFFFFP-8UHK
#define __SFRACT_EPSILON__ 0x1P-7HR
#define __FLT_HAS_QUIET_NAN__ 1
#define __FLT_MAX_10_EXP__ 38
#define __LONG_MAX__ 0x7fffffffL
#define __DEC128_SUBNORMAL_MIN__ 0.000000000000000000000000000000001E-6143DL
#define __FLT_HAS_INFINITY__ 1
#define __unix 1
#define __USA_FBIT__ 16
#define __UINT_FAST16_TYPE__ unsigned int
#define __DEC64_MAX__ 9.999999999999999E384DD
#define __ARM_32BIT_STATE 1
#define __INT_FAST32_WIDTH__ 32
#define __CHAR16_TYPE__ short unsigned int
#define __PRAGMA_REDEFINE_EXTNAME 1
#define __SIZE_WIDTH__ 32
#define __INT_LEAST16_MAX__ 0x7fff
#define __DEC64_MANT_DIG__ 16
#define __INT64_MAX__ 0x7fffffffffffffffLL
#define __UINT_LEAST32_MAX__ 0xffffffffU
#define __SACCUM_FBIT__ 7
#define __FLT32_DENORM_MIN__ 1.4012984643248171e-45F32
#define __GCC_ATOMIC_LONG_LOCK_FREE 1
#define __SIG_ATOMIC_WIDTH__ 32
#define __INT_LEAST64_TYPE__ long long int
#define __ARM_FEATURE_CLZ 1
#define __INT16_TYPE__ short int
#define __INT_LEAST8_TYPE__ signed char
#define __STDC_VERSION__ 201112L
#define __SQ_FBIT__ 31
#define __DEC32_MAX_EXP__ 97
#define __ARM_ARCH_ISA_THUMB 1
#define __INT_FAST8_MAX__ 0x7f
#define __ARM_ARCH 5
#define __INTPTR_MAX__ 0x7fffffff
#define __QQ_FBIT__ 7
#define linux 1
#define __UTA_IBIT__ 64
#define __FLT64_HAS_QUIET_NAN__ 1
#define __FLT32_MIN_10_EXP__ (-37)
#define __FLT32X_DIG__ 15
#define __LDBL_MANT_DIG__ 53
#define __SFRACT_FBIT__ 7
#define __SACCUM_MIN__ (-0X1P7HK-0X1P7HK)
#define __DBL_HAS_QUIET_NAN__ 1
#define __FLT64_HAS_INFINITY__ 1
#define __SIG_ATOMIC_MIN__ (-__SIG_ATOMIC_MAX__ - 1)
#define __INTPTR_TYPE__ int
#define __UINT16_TYPE__ short unsigned int
#define __WCHAR_TYPE__ unsigned int
#define __SIZEOF_FLOAT__ 4
#define __USQ_FBIT__ 32
#define __pic__ 2
#define __UINTPTR_MAX__ 0xffffffffU
#define __INT_FAST64_WIDTH__ 64
#define __DEC64_MIN_EXP__ (-382)
#define __ULLACCUM_IBIT__ 32
#define __FLT32_DECIMAL_DIG__ 9
#define __INT_FAST64_MAX__ 0x7fffffffffffffffLL
#define __GCC_ATOMIC_TEST_AND_SET_TRUEVAL 1
#define __FLT_DIG__ 6
#define __FLT32_HAS_INFINITY__ 1
#define __UINT_FAST64_TYPE__ long long unsigned int
#define __INT_MAX__ 0x7fffffff
#define __LACCUM_FBIT__ 31
#define __USACCUM_MIN__ 0.0UHK
#define __UHA_IBIT__ 8
#define __INT64_TYPE__ long long int
#define __FLT_MAX_EXP__ 128
#define __UTQ_IBIT__ 0
#define __DBL_MANT_DIG__ 53
#define __INT_LEAST64_MAX__ 0x7fffffffffffffffLL
#define __GCC_ATOMIC_CHAR16_T_LOCK_FREE 1
#define __DEC64_MIN__ 1E-383DD
#define __WINT_TYPE__ unsigned int
#define __UINT_LEAST32_TYPE__ unsigned int
#define __SIZEOF_SHORT__ 2
#define __ULLFRACT_IBIT__ 0
#define __LDBL_MIN_EXP__ (-1021)
#define __arm__ 1
#define __FLT64_MAX__ 1.7976931348623157e+308F64
#define __UDA_IBIT__ 32
#define __WINT_WIDTH__ 32
#define __INT_LEAST8_MAX__ 0x7f
#define __FLT32X_MAX_10_EXP__ 308
#define __LFRACT_FBIT__ 31
#define __LDBL_MAX_10_EXP__ 308
#define __ATOMIC_RELAXED 0
#define __DBL_EPSILON__ ((double)2.2204460492503131e-16L)
#define __UINT8_C(c) c
#define __FLT64_MAX_EXP__ 1024
#define __INT_LEAST32_TYPE__ int
#define __SIZEOF_WCHAR_T__ 4
#define __UINT64_TYPE__ long long unsigned int
#define __LLFRACT_MAX__ 0X7FFFFFFFFFFFFFFFP-63LLR
#define __TQ_FBIT__ 127
#define __INT_FAST8_TYPE__ signed char
#define __ULLACCUM_EPSILON__ 0x1P-32ULLK
#define __UHQ_IBIT__ 0
#define __LLACCUM_IBIT__ 32
#define __FLT64_HAS_DENORM__ 1
#define __FLT32_EPSILON__ 1.1920928955078125e-7F32
#define __DBL_DECIMAL_DIG__ 17
#define __STDC_UTF_32__ 1
#define __INT_FAST8_WIDTH__ 8
#define __DEC_EVAL_METHOD__ 2
#define __FLT32X_MAX__ 1.7976931348623157e+308F32x
#define __TA_FBIT__ 63
#define __UDQ_IBIT__ 0
#define __ORDER_BIG_ENDIAN__ 4321
#define __ACCUM_EPSILON__ 0x1P-15K
#define __UINT32_C(c) c ## U
#define __INTMAX_MAX__ 0x7fffffffffffffffLL
#define __BYTE_ORDER__ __ORDER_LITTLE_ENDIAN__
#define __FLT_DENORM_MIN__ 1.4012984643248171e-45F
#define __LLFRACT_IBIT__ 0
#define __INT8_MAX__ 0x7f
#define __LONG_WIDTH__ 32
#define __PIC__ 2
#define __UINT_FAST32_TYPE__ unsigned int
#define __CHAR32_TYPE__ unsigned int
#define __FLT_MAX__ 3.4028234663852886e+38F
#define __USACCUM_FBIT__ 8
#define __INT32_TYPE__ int
#define __SIZEOF_DOUBLE__ 8
#define __FLT_MIN_10_EXP__ (-37)
#define __UFRACT_EPSILON__ 0x1P-16UR
#define __FLT64_MIN__ 2.2250738585072014e-308F64
#define __INT_LEAST32_WIDTH__ 32
#define __INTMAX_TYPE__ long long int
#define __DEC128_MAX_EXP__ 6145
#define __FLT32X_HAS_QUIET_NAN__ 1
#define __ATOMIC_CONSUME 1
#define __GNUC_MINOR__ 3
#define __INT_FAST16_WIDTH__ 32
#define __UINTMAX_MAX__ 0xffffffffffffffffULL
#define __DEC32_MANT_DIG__ 7
#define __FLT32X_DENORM_MIN__ 4.9406564584124654e-324F32x
#define __HA_FBIT__ 7
#define __DBL_MAX_10_EXP__ 308
#define __LDBL_DENORM_MIN__ 4.9406564584124654e-324L
#define __INT16_C(c) c
#define __STDC__ 1
#define __PTRDIFF_TYPE__ int
#define __LLFRACT_MIN__ (-0.5LLR-0.5LLR)
#define __ATOMIC_SEQ_CST 5
#define __DA_FBIT__ 31
#define __UINT32_TYPE__ unsigned int
#define __FLT32X_MIN_10_EXP__ (-307)
#define __UINTPTR_TYPE__ unsigned int
#define __USA_IBIT__ 16
#define __DEC64_SUBNORMAL_MIN__ 0.000000000000001E-383DD
#define __ARM_EABI__ 1
#define __DEC128_MANT_DIG__ 34
#define __LDBL_MIN_10_EXP__ (-307)
#define __SIZEOF_LONG_LONG__ 8
#define __ULACCUM_EPSILON__ 0x1P-32ULK
#define __SACCUM_IBIT__ 8
#define __GCC_ATOMIC_LLONG_LOCK_FREE 1
#define __FLT32X_MIN__ 2.2250738585072014e-308F32x
#define __LDBL_DIG__ 15
#define __FLT_DECIMAL_DIG__ 9
#define __UINT_FAST16_MAX__ 0xffffffffU
#define __GCC_ATOMIC_SHORT_LOCK_FREE 1
#define __INT_LEAST64_WIDTH__ 64
#define __ULLFRACT_MAX__ 0XFFFFFFFFFFFFFFFFP-64ULLR
#define __UINT_FAST8_TYPE__ unsigned char
#define __USFRACT_EPSILON__ 0x1P-8UHR
#define __ULACCUM_FBIT__ 32
#define __ARM_FEATURE_DSP 1
#define __QQ_IBIT__ 0
#define __ATOMIC_ACQ_REL 4
#define __ATOMIC_RELEASE 3
```

**Comment by samueloph** [comment]
Waiting for the experimental build with https://github.com/aircrack-ng/aircrack-ng/commit/149929109eab1f79c4b90aa423f6d6eb4ee0e882 applied to see if it fix. Thanks!

**Comment by samueloph** [comment]
The build went ok, we are ready for an unstable upload :)

https://buildd.debian.org/status/package.php?p=aircrack-ng&suite=experimental

@sbrun @LocutusOfBorg

**Comment by ghost** [comment]
As you can see in my pull request, I got the same error in Void linux. :unamused: 

And I just found another problem...

Using the official tarball:
https://download.aircrack-ng.org/aircrack-ng-1.2.tar.gz
I can run `./autogen.sh`

But... using your github tarball:
https://github.com/aircrack-ng/aircrack-ng/archive/1.2.tar.gz
`./autogen.sh` **fails !**

````
/usr/bin/m4: m4_esyscmd subprocess failed: Bad address
/usr/bin/m4:configure.ac:43: cannot run command `$(pwd)/evalrev || cat $(pwd)/VERSION': Bad address
/usr/bin/m4: m4_esyscmd subprocess failed: Bad address
/usr/bin/m4:configure.ac:43: cannot run command `$(pwd)/evalrev || cat $(pwd)/VERSION': Bad address
/usr/bin/m4: m4_esyscmd subprocess failed: Bad address
/usr/bin/m4:configure.ac:43: cannot run command `$(pwd)/evalrev || cat $(pwd)/VERSION': Bad address
/usr/bin/m4: m4_esyscmd subprocess failed: Bad address
/usr/bin/m4:configure.ac:43: cannot run command `$(pwd)/evalrev || cat $(pwd)/VERSION': Bad address
/usr/bin/m4: m4_esyscmd subprocess failed: Bad address
/usr/bin/m4:configure.ac:43: cannot run command `$(pwd)/evalrev || cat $(pwd)/VERSION': Bad address
/usr/bin/m4: m4_esyscmd subprocess failed: Bad address
/usr/bin/m4:configure.ac:60: cannot run command `$(pwd)/evalrev || cat $(pwd)/VERSION': Bad address
/usr/bin/m4: m4_esyscmd subprocess failed: Bad address
/usr/bin/m4:configure.ac:169: cannot run command `$(pwd)/evalrev || cat $(pwd)/VERSION': Bad address
/usr/bin/m4: m4_esyscmd subprocess failed: Bad address
/usr/bin/m4:configure.ac:169: cannot run command `$(pwd)/evalrev || cat $(pwd)/VERSION': Bad address
autom4te: /usr/bin/m4 failed with exit status: 1
aclocal: error: echo failed with exit status: 1
````
Please could you explain to me **why** it fails?
Regards.

**Comment by aircrack-ng** [comment]
It's a different issue, could you open a new one with this as well as more details on the system you're using to build?

**Comment by ghost** [comment]
I build / cross-build packages every day for more than a year.
Our build system is probably the best around.
All the work is done in a chroot.
GCC 7.3.0.
Also, we use [musl](https://wiki.musl-libc.org/) libc, which is very picky (POSIX).
As I said before, using the github tarball it fails to build.
Using the official tarball, however, it works... at least for x86_64 and i686...
But it fails to build when cross-compiling for ARM 32-bit, as @sbrun pointed out.
Weird.

**Comment by jbenden** [comment]
The error message relates to pipes failing, which is probably the shells to `evalrev` or `cat`. Is it possible for you to attempt to `autoreconf` outside of the chroot, then attempt to proceed to `./configure` inside of the chroot? I am wondering if missing Git or other `evalrev` programs are causing the resulting pipe failure within the chroot.

Thanks,
-Joe

**Comment by ghost** [comment]
@aircrack-ng ok, I don't know what was wrong the last time, I will open a new issue.
